.github/workflows/regression-tests.yml

```diff
@@ -140,6 +140,7 @@ jobs:
     - name: Set up GHDL (Ubuntu)
       if: startsWith(matrix.os, 'ubuntu') && matrix.sim == 'ghdl'
       run: |
+        sudo apt update
```

Comment:
Jesus, it doesn't run update or use newer images with updated caches when starting the instance? Lol.

Reply:
Yes